app/index.js

```diff
@@ -1,5 +1,6 @@
+import 'react-app-polyfill/ie11';
```

Comment:
I'm quite sure that there will be some broken styles in IE, have checked it out?

Reply:
There are some broken stuffs in IE, but in this PR the task is to only display the warning message to IE users. There is another task for fixing stuffs in IE.